_posts/2025-03-06-Consistency-Check.md

```diff
@@ -0,0 +1,83 @@
+---
+title: JabRef's new GUI for Bibliography Consistency Check
+author: priyanshu
+---
+
+Hi, I’m Priyanshu, an engineering student in Computer Science. In this blog post, I’m glad to introduce you to a new feature that I worked on - a graphical user interface (GUI) for checking bibliography consistency in JabRef.
```

Comment:
Please do not use "GUI". I thought, I worked on the in my updates?

As far as I understand the term "GUI", it covers the complete application (also backed by https://en.wikipedia.org/wiki/Graphical_user_interface). We also advertise JabRef as GUI for bibtex.

My proposal was to use "Window" or "Modal Windows". I think, "Wizard" is not good, because the functionality is a single window without a guidance.